layout-server/src/test/java/org/finos/vuu/layoutserver/integration/LayoutIntegrationTest.java

```diff
@@ -377,6 +376,7 @@ void deleteLayout_invalidId_returns400() throws Exception {
     }
 
     private Layout createDefaultLayoutInDatabase() {
+        UUID id = UUID.randomUUID();
```

Comment:
I wonder if it's worth choosing something repeatable for test purposes? Might make things a bit more stable.